src/app/onprem/license-nag-message-management/nag-message.service.js

```diff
@@ -25,11 +25,11 @@ export class NagMessageService {
    * Default constructor that is using resource injection
    * @ngInject for Dependency injection
    */
-  constructor($document, $compile, cheBodyInjectorSvc) {
```

Comment:
if you remove the use of this service then you need to remove this service as you were the one to use it